compose-destinations/src/main/java/com/ramcosta/composedestinations/result/ResultBackNavigator.kt

```diff
@@ -32,8 +32,15 @@ interface ResultBackNavigator<R> {
      *
      * Check [com.ramcosta.composedestinations.result.ResultRecipient] to see
      * how to get the result.
+     *
+     * @param onlyIfResumed if true, will ignore the navigation action if the current `NavBackStackEntry`
+     * is not in the RESUMED state. This avoids duplicate navigation actions.
+     * By default is false to have the same behaviour as [NavController].
      */
-    fun navigateBack(result: R)
+    fun navigateBack(
+        result: R,
+        onlyIfResumed: Boolean = false
+    )
```

Comment:
I believe, if we do this, it also makes sense to do it in the `navigateBack` with no result. What do you think?

Reply:
Sure, I added there also :)